cheroot/test/test_cli.py

```diff
@@ -0,0 +1,81 @@
+"""Tests to verify the command line interface."""
+# -*- coding: utf-8 -*-
+# vim: set fileencoding=utf-8 :
+import sys
+
+import six
+import pytest
+
+from cheroot.cli import (
+    Application,
+    parse_wsgi_bind_addr,
+)
+
+
+@pytest.mark.parametrize(
+    'raw_bind_addr, expected_bind_addr', (
+        ('192.168.1.1:80', ('192.168.1.1', 80)),
+        ('[::1]:8000', ('::1', 8000)),
```

Comment:
This reminds me that we should probably also test catch-all addrs like
```suggestion
        ('[::1]:8000', ('::1', 8000)),
        ('[::]:8000', ('::', 8000)),
        ('12345', (12345)),
```

Reply:
Yeah... I was just avoiding doing the extra work of thinking of future possible uses cases and all the possible values that urllib may support.